src/views/createEditCommunity/__tests__/create.test.tsx

```diff
@@ -925,31 +952,30 @@ describe('create community', () => {
     });
 
     // TODO: this test is important but it's failling for unknown reasons
-    // test('change country', async () => {
-    //     const { getByLabelText, queryAllByTestId, getByA11yLabel } = render(
-    //         <WrappedCreateCommunityScreen />
-    //     );
-    //     await act(async () => {});
-
-    //     fireEvent.press(getByLabelText(i18n.t('generic.country')));
-    //     fireEvent.changeText(getByA11yLabel(i18n.t('generic.search')), 'Port');
-    //     await act(async () => expect(getByLabelText('PT')));
-    //     fireEvent.press(getByLabelText('PT'));
-
-    //     expect(queryAllByTestId('selected-value')[0].children).toContain(
-    //         '🇵🇹 Portugal'
-    //     );
-
-    //     await act(async () => {});
-    //     fireEvent.press(getByLabelText(i18n.t('generic.country')));
-    //     fireEvent.changeText(getByA11yLabel(i18n.t('generic.search')), 'Ang');
-    //     await act(async () => expect(getByLabelText('AO')));
-    //     fireEvent.press(getByLabelText('AO'));
-
-    //     expect(queryAllByTestId('selected-value')[0].children).toContain(
-    //         '🇦🇴 Angola'
-    //     );
-    // });
+    test('change country', async () => {
+        const { getByLabelText, queryAllByTestId, getByA11yLabel } = render(
+            <WrappedCreateCommunityScreen />
+        );
+        await act(async () => {});
+
+        fireEvent.press(getByLabelText(i18n.t('generic.country')));
+        fireEvent.changeText(getByA11yLabel(i18n.t('generic.search')), 'Port');
+        await act(async () => expect(getByLabelText('PT')));
+        fireEvent.press(getByLabelText('PT'));
+
+        expect(queryAllByTestId('selected-value')[0].children).toContain(
+            '🇵🇹 Portugal'
+        );
+
+        // fireEvent.press(getByLabelText(i18n.t('generic.country')));
+        // fireEvent.changeText(getByA11yLabel(i18n.t('generic.search')), 'Ang');
+        // await act(async () => expect(getByLabelText('AO')));
+        // fireEvent.press(getByLabelText('AO'));
+
+        // expect(queryAllByTestId('selected-value')[0].children).toContain(
+        //     '🇦🇴 Angola'
+        // );
+    });
```

Comment:
The goal for this test is to select a country and then change it, not just select. Leaving it like this makes it confusing, giving the impression that the remaining code on comment was forgotten when it's not. It's better to have it fully on comment.

Reply:
It's fully commented then. Although we'll need to invest time on this and see what's the root cause of the problem.